docs/changelog/changelog.rst

```diff
@@ -21,20 +22,22 @@ Highlights
 
 New Features
 ~~~~~~~~~~~~
-
+- Agent start location and rotation can now be optionally randomized on
+  :meth:`~holodeck.environments.HolodeckEnvironment.reset`. See
+  :ref:`location-randomization`.
+  (`#295 <https://github.com/BYU-PCCL/holodeck/issues/295>`_)
 
 Changes
 ~~~~~~~
 
 
 Bug Fixes
 ~~~~~~~~~
-
+- Fixed UAV blades rotating incorrectly (thanks @sethmnielsen!)
+  (`#331 <https://github.com/BYU-PCCL/holodeck/issues/331>`_)
 
 Known Issues
 ~~~~~~~~~~~~
-- The TurtleAgent does not move consistently between Linux and Windows.
```

Comment:
Why was this removed? We still have this issue. Are we planning on fixing this for 0.3.1?

Reply:
I didn't want to have a long, ever growing list of bugs in this section. It's a known defect with 0.3.0 and will carry on until we fix it